doc/tutorials/hello-elektra.md

```diff
@@ -0,0 +1,82 @@
+# Hello, Elektra
+
+This basic tutorial shows you how to compile and run a very basic  Elektra application. For this tutorial we assume that you installed [Elektra](/doc/INSTALL.md) and [CMake](http://cmake.org) on your machine. We also assume that you work a Unix based OS like Linux or macOS.
+
+1. Create a folder called `Hello` somewhere on your disk
+2. Copy the file `examples/helloElektra.c` to the folder `Hello` you just created
+3. Save a file with the following content
+
+   ```cmake
+   cmake_minimum_required(VERSION 3.0)
```

Comment:
If you want we can also add this as example, so that both files can be copied from examples.

Reply:
I think it is okay as it is. Usually developers are already very skilled at copying and pasting code ☺️.